types/dispatcher.d.ts

```diff
@@ -93,6 +96,12 @@ declare class Dispatcher extends EventEmitter {
 }
 
 declare namespace Dispatcher {
+  export interface ComposedDispatcher {
```

Comment:
```suggestion
  export interface ComposedDispatcher extends Dispatcher {
```

Reply:
I was thinking about doing the same, but not sure; the `ComposedDispatcher` just has a subset of the `Dispatcher` methods, are we ok with that?

Reply:
`ComposedDispatcher` extends `Dispatcher`, so it has all the methods: https://github.com/nodejs/undici/blob/9a56796b3e61713f7f2e44ea8c9622b158697c45/lib/dispatcher/dispatcher.js#L42

Reply:
😓 

Thanks, I overlooked it!